Content.Client/Access/UI/IdCardConsoleBoundUserInterface.cs

```diff
@@ -37,9 +37,14 @@ protected override void Open()
             _window = new IdCardConsoleWindow(this, _prototypeManager, accessLevels) {Title = _entityManager.GetComponent<MetaDataComponent>(Owner.Owner).EntityName};
 
             _window.PrivilegedIdButton.OnPressed += _ => SendMessage(new ItemSlotButtonPressedEvent(PrivilegedIdCardSlotId));
-            _window.TargetIdButton.OnPressed += _ => SendMessage(new ItemSlotButtonPressedEvent(TargetIdCardSlotId));
+            _window.TargetIdButton.OnPressed += _ =>
+            {
+                SendMessage(new LogChangesToIdCardMessage());
```

Comment:
Having a separate event for this is a bad idea because a malicious client can just spam these.

Reply:
Oh cool! Didn't know that.

I'll figure a way to call this function but not from the button itself.